bc_obps/registration/management/commands/check_document_file_status.py

```diff
@@ -58,6 +58,7 @@ def handle(self, *args, **options):
         REPEAT_DELAY = options["repeat_delay"]
 
         logger.info("Starting check_document_file_status")
+        RUN_FOREVER = False
```

Comment:
needed to declare this variable outside the if statement otherwise you can hit an error (as I did)